src/main/java/uk/co/jemos/podam/api/AbstractRandomDataProviderStrategy.java

```diff
@@ -542,8 +542,17 @@ public RandomDataProviderStrategy removeAttributeStrategy(
 	public AttributeStrategy<?> getStrategyForAnnotation(
 			final Class<? extends Annotation> annotationClass) {
 
-		return attributeStrategies.get(annotationClass);
+		if (attributeStrategies.containsKey(annotationClass)) {

```

Comment:
containsKey - get result in searching in map twice. This normally is used in case map can contain nulls. Otherwise
value = map.get(key);
if (null != value) {
  // go for plan B
}
is more efficient

Reply:
changed